kitsune/dashboards/tests/test_models.py

```diff
@@ -38,14 +38,14 @@ def test_visit_count_from_analytics(self, _build_request, close_old_connections)
 PAGEVIEWS_BY_DOCUMENT_RESPONSE = {
     "kind": "analytics#gaData",
     "rows": [
-        ["/en-US/kb/hellỗ", "27"],
```

Comment:
Let's update the urls per app. One PR for one app and avoid adding multiple commits per app

Reply:
I'd ask you to reconsider - this is a test in `dashboards` that fails because of a setting change in the `wiki` urls . 

This PR is for `wiki` url changes, and fixing what those changes break, which is tests in a four other apps. You need the `wiki/urls.py` changes in place so your changed tests will work.